pages/index.js

```diff
@@ -88,10 +80,22 @@ export default function HomePage() {
         height={200}
       />
       <LinkContainer>
-        <StyledLink href="/profile/">Profile</StyledLink>
-        <StyledLink href={`/events/upcoming`}>Upcoming Appointments</StyledLink>
-        <StyledLink href={`/events/latest`}>Latest Appointments</StyledLink>
-        <StyledLink href={`/events/wishlist`}>Your Wishlist</StyledLink>
+        <StyledLink href={`/events/latest`}>
```

Comment:
you don't need the brackets and the template literals:

```suggestion
        <StyledLink href="/events/latest">
```

Reply:
Done